api/src/main/java/javax/enterprise/inject/se/SeContainer.java

```diff
@@ -17,12 +17,19 @@
 
 package javax.enterprise.inject.se;
 
+import java.lang.annotation.Annotation;
+
 import javax.enterprise.inject.Instance;
 import javax.enterprise.inject.spi.BeanManager;
 
 /**
  * Provides access to the current container in Java SE.
  *
+ * <p>
+ * SeContainer implements {@link Instance} and therefore might be used to perform programmatic lookup.
+ * If no qualifier is specified in {@link Instance#select(Class, Annotation...)} method the <code>@Default</code> qualifier is assumed.
```

Comment:
I would rather use "If no qualifier is passed to" in all places... and `{@link #select(Class, Annotation...)}`.